src/main/java/com/example/petbuddybackend/entity/user/Client.java

```diff
@@ -19,7 +19,7 @@ public class Client {
     private String email;
 
     @PrimaryKeyJoinColumn
-    @OneToOne(cascade = CascadeType.MERGE, optional = false)
+    @OneToOne(mappedBy = "client", cascade = CascadeType.MERGE, optional = false)
```

Comment:
czym to sie rozni, jesli po stronie Client zamist User damy mappedBy?

Reply:
Na stacku takie coś znalazłem:

>  The annotation JoinColumn indicates that this entity is the owner of the relationship (that is: the corresponding table has a column with a foreign key to the referenced table), whereas the attribute mappedBy indicates that the entity in this side is the inverse of the relationship, and the owner resides in the "other" entity. This also means that you can access the other table from the class which you've annotated with "mappedBy" (fully bidirectional relationship).

Czyli w skrócie `mappedBy` powinno nie być w ownerze, tylko w tym posiadanym obiekcie

Reply:
jasne, dzieki